### PR TITLE
Change the term `function` to `method` in learn content

### LINKED ISSUE #5913: Wrong terminology usage in learn pages
Double check and correct the wrong terminology in the GraphQL learn page: https://ballerina.io/learn/write-a-graphql-api-with-ballerina/

The page uses the terms "resource function" and "remote function" when it actually should be "resource method" and "remote method".

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Change the term `function` to `method` in learn content.
> Fixes #5913 

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

**Review by mohanvive — DISMISSED** [review]

**Review by anuruddhal — DISMISSED** [review]

**Review by daneshk — DISMISSED** [review]

**Review by gimantha — DISMISSED** [review]

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md [hunk 1]
```diff
@@ -393,10 +393,10 @@ service /employees on new http:Listener(8080) {
 }
 ```
 
-### Create the resource functions
+### Create the resource methods
 
-Within this service, you can define resource functions to provide access to the database. The code snippet below 
-demonstrates a resource function that can be used to create a new employee via a `POST` request.
+Within this service, you can define resource methods to provide access to the database. The code snippet below 
+demonstrates a resource methods that can be used to create a new employee via a `POST` request.
```
Comment (MaryamZi) [inline swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md:399]:
```suggestion
demonstrates a resource method that can be used to create a new employee via a `POST` request.
```

File: swan-lake/featured-use-cases/work-with-data-using-queries-in-ballerina.md [hunk 2]
```diff
@@ -79,9 +79,9 @@ In this code:
 
 ## Filter the data
 
-### Create the `filterCountriesByCases` function
+### Create the `filterCountriesByCases` method
 
-To define a function, which will filter out the records, which have values higher than 100,000 for the `cases` field, add the code below to the `main.bal` file.
+To define a method, which will filter out the records, which have values higher than 100,000 for the `cases` field, add the code below to the `main.bal` file.
```
Comment (MaryamZi) [inline swan-lake/featured-use-cases/work-with-data-using-queries-in-ballerina.md:84]: Incorrect. Not a method.

File: swan-lake/featured-use-cases/work-with-data-using-queries-in-ballerina.md [hunk 3]
```diff
@@ -79,9 +79,9 @@ In this code:
 
 ## Filter the data
 
-### Create the `filterCountriesByCases` function
+### Create the `filterCountriesByCases` method
```
Comment (MaryamZi) [inline swan-lake/featured-use-cases/work-with-data-using-queries-in-ballerina.md:82]:
```suggestion
### Create the `filterCountriesByCases` function
```

File: swan-lake/featured-use-cases/work-with-data-using-queries-in-ballerina.md (hunk 2 above)
Comment (MaryamZi) [inline swan-lake/featured-use-cases/work-with-data-using-queries-in-ballerina.md:84]:
```suggestion
To define a function, which will filter out the records, which have values higher than 100,000 for the `cases` field, add the code below to the `main.bal` file.
```

**Review by MaryamZi — COMMENTED** [review]
Adding suggestions as requested by @praneesha

File: spec/http/spec.md [hunk 4]
```diff
@@ -2018,8 +2019,7 @@ work such as the below.
  - Validating
  - Securing
 
-Interceptors are designed for both request and response flows. There are just service objects which will be executed in
-a configured order to intercept request and response. These interceptor services can only have either a resource method 
+Interceptors are designed for both request and response flows. There are just service objects which will be executed in a configured order to intercept request and response. These interceptor services can only have either a resource method
```
Comment (MaryamZi) [inline spec/http/spec.md:2022]: Why are these lines combined? There seems to have been a specific length.

File: swan-lake/other/network-communication/HTTP/http-services/payload-data-binding.md [hunk 5]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: ballerina-left-nav-pages-swanlake
 title: Payload data binding
-description: The HTTP service resource payloads can be directly data bound to the resource function parameters. 
+description: The HTTP service resource payloads can be directly data bound to the resource mehod parameters.
```
Comment (MaryamZi) [inline swan-lake/other/network-communication/HTTP/http-services/payload-data-binding.md:4]:
```suggestion
description: The HTTP service resource payloads can be directly data bound to the resource method parameters. 
```

**Pull request merged**